### ISSUE TITLE
metasploit: ruby fatal error

### ISSUE THREAD

**Issue by nil0x42** [BlackArch/blackarch#1049]
**`$ /bin/msfconsole`**
```
fatal: Not a git repository (or any of the parent directories): .git
/usr/share/metasploit/modules/exploits/windows/http/easyfilesharing_seh.rb:31: warning: key "DefaultOptions" is duplicated and overwritten on line 43
/usr/share/metasploit/modules/exploits/multi/http/atutor_sqli.rb:246: warning: key :private_type is duplicated and overwritten on line 247
/usr/share/metasploit/modules/payloads/singles/cmd/mainframe/reverse_shell_jcl.rb:247:in `to_s': wrong number of arguments (given 1, expected 0) (ArgumentError)
        from /usr/share/metasploit/modules/payloads/singles/cmd/mainframe/reverse_shell_jcl.rb:247:in `replace_var'
        from /usr/share/metasploit/lib/msf/core/payload.rb:356:in `block in substitute_vars'
        from /usr/share/metasploit/lib/msf/core/payload.rb:352:in `each_pair'
        from /usr/share/metasploit/lib/msf/core/payload.rb:352:in `substitute_vars'
        from /usr/share/metasploit/lib/msf/core/payload.rb:652:in `internal_generate'
        from /usr/share/metasploit/lib/msf/core/payload.rb:312:in `generate'
        from /usr/share/metasploit/lib/msf/core/payload/single.rb:40:in `generate'
        from /usr/share/metasploit/lib/msf/core/payload.rb:200:in `size'
        from /usr/share/metasploit/lib/msf/core/payload_set.rb:91:in `block in recalculate'
        from /usr/share/metasploit/lib/msf/core/payload_set.rb:78:in `each_pair'
        from /usr/share/metasploit/lib/msf/core/payload_set.rb:78:in `recalculate'
        from /usr/share/metasploit/lib/msf/core/modules/loader/base.rb:250:in `block in load_modules'
        from /usr/share/metasploit/lib/msf/core/modules/loader/base.rb:247:in `each'
        from /usr/share/metasploit/lib/msf/core/modules/loader/base.rb:247:in `load_modules'
        from /usr/share/metasploit/lib/msf/core/module_manager/loading.rb:117:in `block in load_modules'
        from /usr/share/metasploit/lib/msf/core/module_manager/loading.rb:115:in `each'
        from /usr/share/metasploit/lib/msf/core/module_manager/loading.rb:115:in `load_modules'
        from /usr/share/metasploit/lib/msf/core/module_manager/module_paths.rb:41:in `block in add_module_path'
        from /usr/share/metasploit/lib/msf/core/module_manager/module_paths.rb:40:in `each'
        from /usr/share/metasploit/lib/msf/core/module_manager/module_paths.rb:40:in `add_module_path'
        from /usr/share/metasploit/lib/msf/base/simple/framework/module_paths.rb:50:in `block in init_module_paths'
        from /usr/share/metasploit/lib/msf/base/simple/framework/module_paths.rb:49:in `each'
        from /usr/share/metasploit/lib/msf/base/simple/framework/module_paths.rb:49:in `init_module_paths'
        from /usr/share/metasploit/lib/msf/ui/console/driver.rb:204:in `initialize'
        from /usr/share/metasploit/lib/metasploit/framework/command/console.rb:62:in `new'
        from /usr/share/metasploit/lib/metasploit/framework/command/console.rb:62:in `driver'
        from /usr/share/metasploit/lib/metasploit/framework/command/console.rb:48:in `start'
        from /usr/share/metasploit/lib/metasploit/framework/command/base.rb:82:in `start'
        from /usr/share/metasploit/msfconsole:48:in `<main>'
```

**Comment by noptrix** [comment]
do you still have the problem?

**Comment by nil0x42** [comment]
I haven't :)
Fixed with version 4:38127.63c6a6d-1
